unified-codes/apps/web/src/components/templates/Explorer.tsx

```diff
@@ -59,17 +87,28 @@ export const ExplorerComponent: Explorer = ({
     onUpdateVariables({ ...variables, page: 0, rowsPerPage });
   };
 
+  const handleSort = (orderBy: string) => {
+    const orderDesc = orderBy === variables?.orderBy ? !variables.orderDesc : false;
+    onUpdateVariables({ ...variables, orderBy, orderDesc });
+  };
+
+  const childProps = {
+    tableProps: { alternatingRowColour: '#f5f5f5', stripedRows: true },
+    rowProps: { rowProps: { style: { backgroundColor: '' } } },
+  };
+
   return (
-    <Grid container justify="center">
-      <EntityBrowser
-        entities={entityData}
-        onChangePage={handleChangePage}
-        onChangeRowsPerPage={handleChangeRowsPerPage}
-        onClear={handleClear}
-        onSearch={handleSearch}
-        variables={variables}
-      />
-    </Grid>
+    <EntityBrowser
+      childProps={childProps}
```

Comment:
Think same issue as `classes` here, I think? Not sure about the parent having a generic prop which handles all the child props, do you think could be better to figure out what props we need to drill through and make them explicit?